src/components/Compounder/CompounderVaultApy.tsx

```diff
@@ -6,7 +6,7 @@ import { useVaultApy, useVaultPoolId } from "@/hooks"
 
 import Skeleton from "@/components/Skeleton"
 
-export const VaultApy: FC<VaultProps> = (props) => {
+export const CompounderVaultApy: FC<VaultProps> = (props) => {
```

Comment:
Rename the VaultApy to CompounderVaultApy, the concentrator can't reuse it because it uses the compounder API endpoints